rosidl_generator_c/test/test_interfaces.c

```diff
@@ -896,3 +941,325 @@ int test_submessages(void)
 
   return 0;
```

Comment:
If think there was a good reason for not doing so. Can't think of it right now and can't test it today. It shouldn't block this PR though